pyresample/test/utils.py

```diff
@@ -166,7 +166,8 @@ def __exit__(self, type, value, traceback):
 
 def create_test_longitude(start, stop, shape, twist_factor=0.0, dtype=np.float32):
     """Get basic sample of longitude data."""
-    if start > stop:
+    if start > stop and start > 0 > stop:
```

Comment:
doesn't the second one imply the first one? ie would it be enough with
`if start > 0 > stop:` ?

Reply:
Probably. This was in the middle of testing and I just wanted to make sure I had the right guess as to what was going wrong. I'll clean it up in a little while.